src/lib/Container.svelte

```diff
@@ -1,11 +1,12 @@
 <script>
   import { setContext } from 'svelte'
-  import { isServer } from './util'
+  import { isServer, register } from './util'
 
   export let stripe
 
   const elements = isServer ? null : stripe.elements()
 
+  register(stripe)
```

Comment:
@joshnuss should I also add `onMount` here? Or is it good enough to just check `!isServer`? 

Happy to follow whichever approach you prefer.

Reply:
Yeah, I think it's good enough to just check `!isServer`